monkestation/code/modules/storytellers/antag_rep/helper_procs.dm

```diff
@@ -0,0 +1,62 @@
+GLOBAL_LIST_INIT(blessed_ckeys, list(
+	"taocat" = list(3, 25),
+)) //this is a lmao moment should be a json but its being left here because lol it goes ckey = list(multiplier, base)
+
+///adjusts antag rep by {VALUE} keeping the value above 0
+/datum/preferences/proc/adjust_antag_rep(value, multiplier = TRUE)
+	if(multiplier)
+		value *= return_rep_multiplier()
+	log_antag_rep("[parent]'s antag rep was adjusted by [value]")
+	antag_rep += value
+	if(antag_rep < 1)
+		log_antag_rep("[parent]'s antag rep was adjusted below 1 resetting to 1")
+		antag_rep = 1
+	save_preferences()
+
+/datum/preferences/proc/reset_antag_rep()
+	var/default = return_default_antag_rep()
+	log_antag_rep("[parent]'s antag rep was reset to default ([default])")
+	antag_rep = default
+	save_preferences()
+
+/datum/preferences/proc/return_default_antag_rep()
+	if(!parent)
+		return 10
```

Comment:
These return values should also use defines, also if these have the same value then I think it could just use `||` and or `?.`